rosbag2_test_common/include/rosbag2_test_common/temporary_directory_fixture.hpp

```diff
@@ -31,7 +33,14 @@ class TemporaryDirectoryFixture : public Test
 public:
   TemporaryDirectoryFixture()
   {
-    temporary_dir_path_ = rcpputils::fs::create_temporary_directory("tmp_test_dir_").string();
+    std::filesystem::path parent_path = std::filesystem::path("/tmpfs");
+    if (!std::filesystem::exists(std::filesystem::path("/tmpfs"))) {
```

Comment:
```suggestion
    if (!std::filesystem::exists(parent_path))) {
```

Reply:
nice catch :sweat_smile:

Reply:
Fixed in the https://github.com/ros2/rosbag2/pull/1901/commits/8fb8c6ee1b14208625f915a3f9940f0c2c8a330d commit.

Reply:
@fujitatomoya Friendly ping here to verify changes and get approval.

Reply:
Since you applied Tomoya's suggestion as-is, I think we can spare Tomoya the time to look at this again and just proceed with the PR :grin:

Reply:
..although I assumed that, by re-requesting a review from Tomoya, it would remove the "change requested" flag, but apparently that's not true.